deployment/docker-compose.yml

```diff
@@ -61,11 +60,23 @@ uwsgi:
     - ./media:/home/web/media:rw
     - ./reports:/home/web/reports
     - ./logs:/var/log/
+  user: root
+
+uwsgi-ephemeral:
+  extends: uwsgi-base
+  container_name: mezzanine-uwsgi-ephemeral
```

Comment:
Let's remove the container_name, because there might be more than one at any given time.